src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -92,6 +92,16 @@ function refinementCommandCancellationFn(cmd) {
     return !cmd.requester.isDisplayed();
 }
 
+function checkNodeElevationTextureValidity(texture, noDataValue) {
+    // We check if the elevation texture has some significant values through corners
+    const tData = texture.image.data;
+    const l = tData.length;
+    return tData[0] > noDataValue &&
+           tData[l - 1] > noDataValue &&
+           tData[Math.sqrt(l) - 1] > noDataValue &&
```

Comment:
I'd prefer `tData[width - 1]` and `tData[height * width]` instead of `Math.sqrt` usage but it's up to you